src/logger.js

```diff
@@ -0,0 +1,26 @@
+import colors from 'colors/safe';
+
+var colorList = ['blue', 'yellow', 'magenta', 'cyan'];
+
+var enabled = !!process.env.DEBUG;
+
+export default function logger (command, ...messages) {
+  if (enabled) {
+    var color = colorList[command.length % colorList.length];
```

Comment:
I don't get the point of these color calls.  Can you explain?

To me it looks like we are almost randomly choosing a color for the given console message